en/messages.json

```diff
@@ -113,6 +121,9 @@
     "popupSignUpPanelContinue": {
         "message": "Continue"
     },
+    "popupSignUpPanelSignIn": {
+        "message": "Sign in"
+    },
     "popupRemainingAliases_2": {
```

Comment:
`aliases`-> `masks` in the string and the description

Reply:
I'm not sure which string this comment is referring to.